test/integration/full-content/full-content.spec.js

```diff
@@ -197,25 +191,29 @@ describe( 'full post content fixture', () => {
 			// The `core/template` is not worth testing here because it's never saved, it's covered better in e2e tests.
 			.filter( ( name ) => name.indexOf( 'core-embed' ) !== 0 && name !== 'core/template' )
 			.forEach( ( name ) => {
-				const nameToFilename = name.replace( /\//g, '__' );
-				const foundFixtures = fileBasenames
-					.filter( ( basename ) => (
-						basename === nameToFilename ||
-						startsWith( basename, nameToFilename + '__' )
+				const nameToFilename = blockNameToFixtureBaseName( name );
+				const foundFixtures = blockBaseNames
+					.filter( ( baseName ) => (
```

Comment:
It looks like `basename` is a thing: https://en.wikipedia.org/wiki/Basename

It might be fine to leave it as it was. @aduth @nerrad or @adamsilverstein? What do you think?

Reply:
If you're asking whether `basename` should be left used instead of `baseName` I would agree, it does mean a number of other new code introduced in this pull that would need updated as well so that `basename` is used consistently.

In the same vein, `filename` should probably be used instead of `fileName` (https://en.wikipedia.org/wiki/Filename).

Reply:
English is hard when it meets computer science :)

Reply:
heh, English is _hard_ period (even to native speakers).